app/workers/zync_worker.rb

```diff
@@ -54,6 +54,8 @@ def find_model(name)
 
   class UnprocessableEntityError < InvalidResponseError; end
 
+  Bugsnag::Middleware::ClassifyError::INFO_CLASSES << UnprocessableEntityError.to_s
```

Comment:
Hmm can't we do that in a config file?

Reply:
Sorry, I was just following what we have in the file: https://github.com/3scale/porta/blob/795ec0d97ee5893da92e1744440227bb8fb953cf/app/workers/zync_worker.rb#L192

Reply:
See https://docs.bugsnag.com/platforms/ruby/sidekiq/customizing-error-reports/#the-report-object